docs/api-constructor.md

```diff
@@ -127,6 +139,29 @@ await sharp({
 }).toFile('noise.png');
 ```
 
+```javascript
+// Generate an image from text
+await sharp({
+  text: {
+    text: 'Hello, world!',
+    width: 400, // max width
+    height: 300 // max height
+  }
+}).toFile('text_bw.png');
+```
+
+```javascript
+// Generate a color image from text using pango markup and font
```

Comment:
Thank you for adding examples, these are great. May I suggest we replace `color` with `RGBA` in the comment and filename to avoid the English spelling conundrum.

Reply:
Done.
I'm not a native english speaker so learned a new word (conundrum) :-)